packages/api/src/external/fhir/document/index.ts

```diff
@@ -604,7 +604,9 @@ function convertCWAdressToFHIR(address: Contained["address"] | undefined): Addre
  */
 function convertCWGenderToFHIR(genders: Gender[] | null | undefined): PatientFHIR["gender"] {
   if (genders && genders[0]) {
-    switch (genders[0].code) {
+    switch (
+      (genders[0] as unknown as { code: string }).code // TODO ENG-200 - Remove the casting
```

Comment:
had to do these b/c was getting a compilation error I couldn't fix after a few attempts, it was becoming a rabbit hole.